documents-webapp/src/main/webapp/vue-app/documents-extensions/components/DocumentsFavoriteItem.vue

```diff
@@ -0,0 +1,75 @@
+<template>
+  <v-list-item class="clickable" @click="openPreview">
+    <v-list-item-icon class="me-3 my-auto">
+      <v-icon size="22" class="icon-default-color"> fas fa-folder-open </v-icon>
+    </v-list-item-icon>
+
+    <v-list-item-content>
+      <v-list-item-title class="text-color body-2">{{ documentTitle }}</v-list-item-title>
+    </v-list-item-content>
+
+    <v-list-item-action>
+      <v-btn icon>
+        <v-icon class="yellow--text text--darken-2" size="18">fa-star</v-icon>
+      </v-btn>
+    </v-list-item-action>
```

Comment:
IMO, this should be added as a common button, out of register component. 
[https://github.com/Meeds-io/social/blob/develop/webapp/portlet/src/main/webapp/vue-apps/common/components/FavoriteButton.vue](url) component to use

Reply:
done